### DISCUSSION TITLE
支持vue吗？

### DISCUSSION THREAD

**Discussion by aMiing** [antvis/XFlow#61] [Q&A] [answered]
有计划支持在vue中使用吗？

**Comment by lyn-boyu** [comment, marked as answer]
暂无计划

**Comment by kolabean** [comment]
你好，是整个XFlow都不支持vue么

**Comment by lloydzhou** [comment]
# 推荐一个专注于管理x6数据的hooks：
https://github.com/lloydzhou/x6-hooks
![image](https://user-images.githubusercontent.com/1826685/219848583-998e03c5-2b5b-42fb-a104-8450d4a00b5f.png)

## 特性
1. 体积小
2. 性能好：内部实现diff/patch使用batchUpdate增量更新画布
3. 同时支持react/vue

## 集成
可以搭配React版本的[Graph](https://github.com/lloydzhou/antv-x6-react-practice/blob/next/src/lib/Graph.tsx)组件或者vue版本的[Graph](https://github.com/lloydzhou/antv-x6-vue-practice/blob/master/src/lib/Graph.tsx)组件使用。

```
<Graph grid resizing snapline keyboard clipboard width={800} height={600}>
  <GraphBehavior />
</Graph>
```
